.evergreen/tox.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -o errexit  # Exit the script with error if any of the commands fail
+
+PYTHON_BINARY="$1"
+
+. .evergreen/utils.sh
+
+if $PYTHON_BINARY -m tox --version; then
+    run_tox() {
+      $PYTHON_BINARY -m tox "$@"
+    }
+elif python3 -m tox --version; then
+    run_tox() {
+      python3 -m tox "$@"
+    }
+elif tox --version; then
+    run_tox() {
+      tox "$@"
+    }
+else # No toolchain present, set up venv before installing tox
+    createvirtualenv PYTHON_BINARY env
```

Comment:
PYTHON_BINARY -> "$PYTHON_BINARY"

Also need to run `python -m pip install tox`